Dockerfile

```
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0
+RUN mkdir /usr/agent
+ARG AGENT_ZIP_FILE
+COPY ${AGENT_ZIP_FILE} /usr/agent/apm-dotnet-agent.zip
+
+RUN apt-get update
+RUN apt-get install unzip
```

nit: reduce size and verbose output

```suggestion
RUN apt-get -qq update
RUN apt-get -qq -y --no-install-recommends install unzip \
    && rm -rf /var/lib/apt/lists/*
```